docs/setup-subrepository.md

```diff
@@ -0,0 +1,178 @@
+];
+
+$client->subrepositories()->mirroredRepositories()->add($subrepositoryName, [$mirroredRepositoryData]);
+```
+
+### Adding packages
+
+Existing packages can be added on the subrepository's packages page via "Add Package" -> "Organization Packages".
+Select all necessary packages and add them to the subrepository. If you need to add additional packages to a subrepository
+that do not yet exist in the organization you can add them using the "Add Package" dialog and the previously set up credentials.
+
+Mirrored packages will automatically be added to your subrepository on use. We do not recommend manually adding them to your subrepository.
+
+Packages can also be added via our API using our [API client](https://github.com/packagist/private-packagist-api-client) with the following code snippet:
+
+```
+<?php
+
+require_once __DIR__ . '/vendor/autoload.php';
+
+$subrepositoryName = 'company-blog';
+$url = 'https://github/acme/company-blog-design';
+
+$client = new \PrivatePackagist\ApiClient\Client();
+$client->authenticate('api-token', 'api-secret');
+$client->subrepositories()->packages()->createVcsPackage($subrepositoryName, $url);
+```
+
+### Granting members access
+
+Permissions in a subrepository are managed via team, the same way permissions are managed in your organization.
+You can add teams to a subrepository and all members of that team will automatically be granted access to it.
+Team membership changes e.g. through a synchronization will automatically be applied to subrepositories as well.
+
+One notable difference in subrepositories is that all members automatically have access to all packages inside a subrepository.
+You do not have to manually grant access to packages.
+
+Teams can also be added to a subrepository via our API using our [API client](https://github.com/packagist/private-packagist-api-client) with the following code snippet:
+
+```
+<?php
+
+require_once __DIR__ . '/vendor/autoload.php';
+
+$subrepositoryName = 'company-blog';
+
+$client = new \PrivatePackagist\ApiClient\Client();
+$client->authenticate('api-token', 'api-secret');
+
+$teams = $client->teams()->all($subrepositoryName, $url);
+// find id of team to be added
+$teamId = array_pop($teams)['id'];
+
+$team = [
+  'id' => $teamId,
+  'permission' => 'view',
+];
+
+$client->subrepositories()->addOrEditTeams($subrepositoryName, [$team]);
+```
+
+### Granting external collaborators access to a subrepository
+
+Working on a Composer project with external developers might sometimes require that those developers get access to a subrepository
+to make sure they can run composer install and update and help manage the project's dependencies. Adding them as subrepository
+collaborators will only grant them access to selected subrepositories and not give them access to you your Private Packagist organization.
+
+You can invite via email subrepository collaborators on the subrepository's team page via the "Manage Collaborators" button.
```

Comment:
```suggestion
Working on a Composer project with external developers might sometimes require that they get access to a subrepository
to make sure they can run composer install and update and help manage the project's dependencies. Adding them as subrepository
collaborators will only grant them access to selected subrepositories and will not give them access to you your Private Packagist organization.

You can invite subrepository collaborators via email on the subrepository's team page via the "Manage Collaborators" button.
```

Reply:
No snippet for API use here?

Reply:
There is no API for this

Reply:
Guess we generally don't do user management through API so far, so can leave this until it becomes necessary.